js/components/address/isValidAddress.js

```diff
@@ -0,0 +1,53 @@
+function isValidAddress(address) {
+    let errors = [];
+
+    if (typeof address.active !== 'boolean') {
+        errors.push('ERROR: paslaugos aktyvumo statusas turi buti boolean');
+    }
+
+    if (typeof address.icon !== 'string') {
+        errors.push('ERROR: paslaugos icon turi buti tekstinis');
+    }
+    else {
+        if (address.icon === '') {
+            errors.push('ERROR: icon pavadinimas negali buti tuscias.');
+        }
+    }
+
+    if (typeof address.title !== 'string') {
+        errors.push('ERROR: paslaugos pavadinimas turi buti tekstinis');
+    }
+    else {
+        if (address.title === '') {
+            errors.push('ERROR: paslaugos pavadinimas negali buti tuscias.');
+        }
+        if (address.title.length > 20) {
+            errors.push('ERROR: paslaugos pavadinimas yra per ilgas.');
+        }
+
+    }
+    if (typeof address.text !== 'string') {
```

Comment:
Antrame stulpelyje address.text yra el. paštas, gal būtų vertinga pakeisti property pavadinimą? Taip pat trūksta validacijų susijusių su el.paštu, pavyzdžiui, dabar vietoj el. pašto galiu įvesti belekokius skaičius ir juos vis tiek spausdina.